README.md

```diff
@@ -37,6 +37,7 @@ itself) to look for plugins.
 | aks-engine                    | [robsonpeixoto/aks-engine](https://gitlab.com/robsonpeixoto/asdf-aks-engine)                      | [![Build Status](https://travis-ci.org/robsonpeixoto/asdf-aks-engine.svg?branch=master)](https://travis-ci.org/robsonpeixoto/asdf-aks-engine)                                          |
 | alp                           | [asdf-community/asdf-alp](https://github.com/asdf-community/asdf-alp)                             | [![Main workflow](https://github.com/asdf-community/asdf-alp/workflows/Main%20workflow/badge.svg)](https://github.com/asdf-community/asdf-alp/actions)                                 |
 | aria2                         | [asdf-community/asdf-aria2](https://github.com/asdf-community/asdf-aria2)                         | [![Build Status](https://github.com/asdf-community/asdf-aria2/workflows/Main%20workflow/badge.svg)](https://github.com/asdf-community/asdf-aria2/actions)                              |
+| argo                          | [sudermanjr/argo](https://github.com/asdf-argo)                                                   | [![Build Status](https://github.com/sudermanjr/asdf-argo/workflows/master/badge.svg?branch=master)](https://github.com/sudermanjr/asdf-argo/actions)                                   |
```

Comment:
It should be like this.
`[sudermanjr/argo](https://github.com/sudermanjr/asdf-argo)`